mod-mosaic/schemas/mosaic_order_request.json

```diff
@@ -0,0 +1,150 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Mosaic Configuration",
```

Comment:
Description and javaName should be updated to Mosaic Order Request

Reply:
updated